supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc

```diff
@@ -1,10 +1,901 @@
+
+[discrete]
+[[batch-subsystem]]
+==== Batch Subsystem (noun)
+*Description*: Use "Batch Subsystem" when referring to the xref:batch-jberet[batch-jberet] subsystem in titles and headings. Capitalize each word. See xref:batch-jberet[batch-jberet] entry for the correct usage in general text.
+
+*Use it*: yes
+
+*Incorrect forms*: Batch subsystem, batch subsystem
+
+*See also*: xref:batch-jberet[batch-jberet]
+
+[discrete]
+[[batch-jberet]]
+==== batch-jberet (noun)
+*Description*: The "batch-jberet" subsystem is used to configure and manage batch jobs. Write in lower case as two words separated by a hyphen. See xref:batch-subsystem[Batch Subsystem] entry for the correct usage in headings.
+
+*Use it*: yes
+
+*Incorrect forms*: Any form using uppercase letters or omitting the hyphen
+
+*See also*: xref:batch-subsystem[Batch Subsystem]
+
+[discrete]
+[[bean-validation-subsystem]]
+==== Bean Validation Subsystem (noun)
+*Description*: Use "Bean Validation Subsystem" when referring to the xref:bean-validation[bean-validation] subsystem in titles and headings. Capitalize each word. See xref:bean-validation[bean-validation] entry for the correct usage in general text.
+
+*Use it*: yes
+
+*Incorrect forms*: Bean Validation subsystem, Bean validation subsystem, bean validation subsystem
+
+*See also*: xref:bean-validation[bean-validation]
+
+[discrete]
+[[bean-validation]]
+==== bean-validation (noun)
+*Description*: The "bean-validation" subsystem is used to configure validation of Java bean object data. Write in lower case as two words separated by a hyphen. See xref:bean-validation-subsystem[Bean Validation Subsystem] entry for the correct usage in headings.
+
+*Use it*: yes
+
+*Incorrect forms*: Any form using uppercase letters or omitting the hyphen
+
+*See also*: xref:bean-validation-subsystem[Bean Validation Subsystem]
+
+[discrete]
+[[built-in-messaging]]
+==== built-in messaging (noun)
+*Description*: "built-in messaging" is an acceptable term for referring to the built-in messaging system in JBoss EAP. Other acceptable terms are "JBoss EAP messaging" and "JBoss EAP built-in messaging".
+
+*Use it*: yes
+
+*Incorrect forms*: ActiveMQ, ActiveMQ Artemis
+
+*See also*: xref:jboss-eap-built-in-messaging[JBoss EAP built-in messaging], xref:jboss-eap-messaging[JBoss EAP messaging]
+
+// ***********************
+// Terms starting with 'C'
+// ***********************
+
+[discrete]
+[[cli-eap]]
+==== CLI (noun)
+*Description*: "CLI" is an abbreviation of "command line interface". Do not use "CLI" by itself to refer to the management CLI.
+
+*Use it*: no
+
+*Incorrect forms*:
+
+*See also*: xref:management-cli[management CLI]
+
+[discrete]
+[[core-management-subsystem]]
+==== Core Management Subsystem (noun)
+*Description*: Use "Core Management Subsystem" when referring to the xref:core-management[core-management] subsystem in titles and headings. Capitalize each word. See xref:core-management[core-management] entry for the correct usage in general text.
```

Comment:
In other RSSG entries, the title casing instructions are included in the main glossary entry instead of being a separate glossary entry. Recommend doing so here as well to reduce the number of entries. Ditto for all subsequent uses.

Reply:
Reworking those entries will be a big task that might need to be handled through a separate pull request in the future.